properties/P000133.md

```diff
@@ -18,6 +18,6 @@ refs:
 
 Linearly ordered topological space (LOTS): a space whose topology can be induced by a total order on the underlying set.
```

Comment:
```suggestion
A space whose topology can be induced by a total order on the underlying set.
```

Reply:
modified your suggestion slightly, so it connects better with the next paragraph